Deep service insight endpoint (#3261)
nginxinc/kubernetes-ingress: docs/content/configuration/global-configuration/command-line-arguments.md

> * If the argument is not set, the prometheus endpoint will not use a TLS connection.

Reviewer: ```suggestion
* If the argument is not set, the Prometheus endpoint will not use a TLS connection.
```

Reviewer: Prometheus is capitalised as a proper noun.

* If the argument is set, but the Ingress Controller is not able to fetch the Secret from Kubernetes API, the Ingress Controller will fail to start.
&nbsp;
<a name="cmdoption-enable-service-insight"></a>

### -enable-service-insight

> Enables exposing Service Insight enpoint for Ingress Controller.

Reviewer: ```suggestion
Exposes the Service Insight endpoint for Ingress Controller.
```

Reviewer: Enabling is an implicit part of exposure.

&nbsp;
<a name="cmdoption-service-insight-listen-port"></a>

### -service-insight-listen-port `<int>`

Sets the port where the Service Insight is exposed.

Format: `[1024 - 65535]` (default `9114`)
&nbsp;
<a name="cmdoption-service-insight-tls-secret"></a>

### -service-insight-tls-secret `<string>`

> A Secret with a TLS certificate and key for TLS termination of the Service Insight enpoint.

Reviewer: ```suggestion
A Secret with a TLS certificate and key for TLS termination of the Service Insight endpoint.
```

> * If the argument is not set, the service insight endpoint will not use a TLS connection.

Reviewer: ```suggestion
* If the argument is not set, the Service Insight endpoint will not use a TLS connection.
```
